### ISSUE TITLE
Problems

### ISSUE THREAD

**Issue by fauks** [rommulaner/Anycubic_Vyper_LCD_CE_6.1d#1]
Both the screen and motherboard update seem to go smoothly. But once I'm at the home screen, it does not respond to any touches. When I restart the machine, it says its stuck and taking to long to load. 

I'm using FAT32 4096 and have tried renaming the MB file with no success. 

Any help would be appreciated!

**Comment by fauks** [comment]
So I have a different screen version. It takes a 4.5 config file and not 3.5. I was able to update that so the screen works now but it still freezes on bootup after the first boot. The first boot with the community firmware seems to work but then it freezes.

**Comment by rommulaner** [comment]
Shure that the MB was flashed OK? Light was flashing after it?
My Vyper was ordered last December directly at Anycubic, so it should be the latest Version I think and there v3.5 is the only one that works for me.

**Comment by fauks** [comment]
> Shure that the MB was flashed OK? Light was flashing after it? My Vyper was ordered last December directly at Anycubic, so it should be the latest Version I think and there v3.5 is the only one that works for me.

I'm positive. My screen uses DGUS2 v4.5. 

The CR-6 repository has instructions and T5L_UI .bin files for both 3.5 and 4.5. I grabbed the file from their and the touch screen works again but the firmware will still not load. 

I just flashed the latest CR-6 DWIN_SET folder with the 4.5 .bin file and the Vyper 6.1d firmware and now I'm able to use the menu but it looks like a CR-6 UI and not the Vyper one. 

Can you help me figure out how to create a 4.5 DWIN_SET files that will work with the Vyper 6.1d MB firmware?

Here is where I found the information about the new screens. 

https://github.com/CR6Community/CR-6-touchscreen#cr-6-touchscreen-firmware

I purchased this printer about a week ago.

**Comment by rommulaner** [comment]
You don't have to create a new DWIN, you just need to use the right cfg files in the folder an flash it. If you create a new DWIN set it is independent of the cfg files.

**Comment by fauks** [comment]
Well what files do I need to replace from the CR-6 DWIN_SET folder?

**Comment by rommulaner** [comment]
Use my DWIN and replace the .cfg file and test it. Make shure that there not 000 on all file on the Boot screen after flashing.
Also when MB is flashed successfully the light is flashing.

**Comment by fauks** [comment]
What about the T5L_UI_V45.BIN file from the CR-6 DWIN_SET folder? That is the main file that fixed the touch screen to start working again.

**Comment by fauks** [comment]
Here are the two folders. 
![image](https://user-images.githubusercontent.com/82473060/177210194-d27d4bb8-80b4-405a-848e-da8c5feb9517.png)

**Comment by fauks** [comment]
> Use my DWIN and replace the .cfg file and test it. Make shure that there not 000 on all file on the Boot screen after flashing. Also when MB is flashed successfully the light is flashing.

When I flash the MB, I hear the beeps and see the main menu and the light flashes once, then it's just solid. If I click anything, it freezes. When I restart, I get stuck on the loading screen.

**Comment by rommulaner** [comment]
I've just uploaded a folder with the two files from the origin Vyper 2.4.5 Version in the DWIN Folder.
CFG_4.5

**Comment by fauks** [comment]
> I've just uploaded a folder with the two files from the origin Vyper 2.4.5 Version in the DWIN Folder. CFG_4.5

Okay. So keep the T5L_UI_V35_*.BIN file from your DWIN_SET? I think the 4.5 version uses a different kernal according to the CR-6 page.

**Comment by fauks** [comment]
I tried both. When updating the MB firmware, I get the beeps and the light starts flashing. After restarting, it says the bootloader is confused and doesn't go to the menu. 

Also when the MB firmware first finishes, all the text is stacked on top of each other.

**Comment by rommulaner** [comment]
Sorry just updated it again. Try some of the BIN files I added.

**Comment by fauks** [comment]
> Sorry just updated it again. Try some of the BIN files I added.

Try the versions like V47? V40?

![image](https://user-images.githubusercontent.com/82473060/177215201-3368bc64-9439-48b6-b596-36d4cb4cb071.png)

This won't brick the screen will it? haha

I appreciate you helping!

**Comment by fauks** [comment]
So far none of them are working.

**Comment by fauks** [comment]
Tried them all, none of them work.

**Comment by fauks** [comment]
This is where I found the V45 file. This file makes the ZombieStudio firmware work and it makes the CR-6 firmware work on the Vyper. The screen works at least, I doubt its actually functional as its made for a CR-6 and not the Vyper. 

https://github.com/CR6Community/CR-6-touchscreen/releases/tag/61F_RC_260522_v1

"Attempt to extend the CR-6-touchscreen firmware to operate with DWIN display hardware flashed with and calibrated to run DGUS2 v4.5 or higher, such as those being sold with new CR6 machines as of approximately Dec 2021."

**Comment by fauks** [comment]
I figured it out. I created a CFG file to recalibrate the screen after flashing your DWIN_SET files. Then flashed the MB firmware, clicked setup, restore defaults, restarted and it finally booted back up into the menu instead of just hanging. Also, the instructions say to restart after flashing the MB then hitting setup - restore defaults. I had do this as soon as the MB was flashed before rebooting to get this to work.
